### ISSUE TITLE
No module named 'trax.supervised.tf_inputs' or Configurable 'train' doesn't have a parameter named 'has_weights'

### ISSUE THREAD

**Issue by ElhamSol** [google/trax#686]
### Description

I'm trying to run the code presented here https://github.com/google/trax/blob/master/trax/models/reformer/machine_translation.ipynb

I replaced this cell
```
# We'll be using a pre-trained reversible transformer-base model.
# First, load the config (which sets all needed hyperparameters).
!gsutil cp gs://trax-ml/reformer/mt/config.gin ./config.gin
gin.parse_config_file('./config.gin')
```
with:
```
!gsutil cp "/content/gdrive/My Drive/work_files/configs/reformer_wmt_ende.gin" ./config.gin
gin.parse_config_file('./config.gin')
```
to load the config from my google drive.
But I get the error "No module named 'trax.supervised.tf_inputs'".
I tried the same with "transformer_lm_wmt_ende_8gb.gin" and had the same error.

![image](https://user-images.githubusercontent.com/63062486/83647239-ad4c1f00-a582-11ea-954a-654ec04d648a.png)

and:
![image](https://user-images.githubusercontent.com/63062486/83657487-eab6a980-a58e-11ea-9836-ee95c6a75ff3.png)

**Comment by ElhamSol** [comment]
It was resolved for reformer when I use trax v1.2.3
For transformer I used gin.parse_config_file('./config.gin', skip_unknown=True) and I don't get the error but is it the correct way?

**Comment by lukaszkaiser** [comment]
Yes. It looks like the problem is using a new config file with an old version of Trax. We're aiming to make a release soon, that should resolve it.
